src/cryptography/x509/base.py

```diff
@@ -1169,5 +1184,69 @@ def build(self, backend: typing.Any = None) -> RevokedCertificate:
         )
 
 
+class Profile(utils.Enum):
+    RFC5280 = 0
+    WebPKI = 1
+
+
+class PolicyBuilder:
+    def __init__(
+        self,
+        *,
+        name: typing.Optional[str] = None,
+        time: typing.Optional[datetime.datetime] = None,
+        profile: Profile = Profile.WebPKI,
+    ):
+        self._name = name
+        self._time = time
+        self._profile = profile
```

Comment:
NB: Unlike the other builder APIs, this one intentionally has public kwargs. Let me know if you're prefer not to do that; I figured that it solves the "some things fixed, other things always change" API distinction problem.

Reply:
No public constructor is how all of our APIs work.

Reply:
f21a4e98a